src/resolver/resolver.ts

```diff
@@ -155,8 +156,8 @@ export function resolveModule(props: IResolverProps): IResolver {
       }
       const pkg = nodeModuleLookup(props, moduleParsed);
 
-      if (pkg.error) {
-        return { error: pkg.error };
+      if ("error" in pkg) {
```

Comment:
any particular reason that was changed?

Reply:
To satisfy TypeScript's type checker.

Here's a quick explanation

![image](https://user-images.githubusercontent.com/158330/77367355-54653c80-6d30-11ea-81d1-f405654ebfe1.png)